### ISSUE TITLE
Addition of callout text/legend

### ISSUE THREAD

**Issue by SteveBowdz** [AUS-DOH-Safety-and-Quality/PowerBI-Funnels#159]
Is it possible to add an option for callout text for each group so groups can be identified when chart is 'printed' for reports etc.

**Comment by andrjohns** [comment]
Could you clarify what you mean by 'callout text'? I haven't come across that before sorry! If you have an existing example of how you'd like it to look, that would also be great.

If it's just displaying the group name instead of the scatter dot, you can already request this (as of version 1.5.2.1):

![image](https://github.com/AUS-DOH-Safety-and-Quality/PowerBI-Funnels/assets/27717896/7a083657-a47a-4e99-b809-e38c8826f911)

**Comment by SteveBowdz** [comment]
Thanks, this is the functionality we are looking for.

However, downloading the plugin from [appsource](https://appsource.microsoft.com/en-au/product/power-bi-visuals/healthdepartmentwa1667894240640.powerbi-funnel-charts?tab=Overview) downloads a file that say PBIFUN1.5.2.1, yet when installing it results in version 1.5.0.2

![image](https://github.com/AUS-DOH-Safety-and-Quality/PowerBI-Funnels/assets/167374003/02587223-3553-408c-9559-ab85141b34d2)

Grateful for any suggestions

**Comment by andrjohns** [comment]
The update to the latest version was only accepted by AppSource early last week, there's usually a delay of ~2 weeks before PowerBI updates the versions that it distributes. If it's not showing for you just yet, then it should update in the next or so

**Comment by SteveBowdz** [comment]
> If I am using the new option/functionality correctly, it replaces each scatter dot with the corresponding annotation. Is it possible to instead place the text next to the dot:
> 
> ![Random Data](https://private-user-images.githubusercontent.com/93962658/336746299-ad814169-6f6d-423b-acc5-86e85ce2d42e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc2MzA1NzQsIm5iZiI6MTcxNzYzMDI3NCwicGF0aCI6Ii85Mzk2MjY1OC8zMzY3NDYyOTktYWQ4MTQxNjktNmY2ZC00MjNiLWFjYzUtODZlODVjZTJkNDJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDIzMzExNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkyODZlZjkzOGU2MDc1MGJhZWFjYzAxNzUxZDk4M2FjMWYzYTlkYzQ2MWNmYWI0NTc1OWVkZTUwNWUxOGY1YjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._nJY1GAvNvHcjYTan_SZfuaju-bX84juPqxI1ZRH0H0)
> 
> Is there an option to show (or not show) outlier labels only, similar to the example above?

Yes, this is sort of the function we would be keen to see. Both the scatter dot AND the name. Would be good if it had the option to turn on/off for outliers or include for all points

**Comment by alex-cplee** [comment]
@andrjohns Is there any chance that you will add new functions to the PBI-Funnels as described above?

**Comment by andrjohns** [comment]
> @andrjohns Is there any chance that you will add new functions to the PBI-Funnels as described above?

Yep, I've worked out a good way to handle labelling in the visuals ([corresponding SPC example](https://github.com/AUS-DOH-Safety-and-Quality/PowerBI-SPC/issues/297#issuecomment-2520486512)), so similar functionality will be coming to the funnels visual in the coming months
